### ISSUE TITLE
Color contrast on select button

### ISSUE THREAD

**Issue by Nealium104** [uklibraries/findingaid#35]
Contrast for the select button between text and background is insufficient. Contrast for normal text should be a 4.5:1. Offending button is created in `app/assets/js/requests.js` starting at line 739. The color is a bootstrap default for `btn-warning`.

<img width="452" height="231" alt="Image" src="https://github.com/user-attachments/assets/27e8dd96-73fa-49b6-82bd-2fd2637aff2b" />

**Comment by nrsa222** [comment]
This looks to be a duplicate issue of #9.
